skyportal/handlers/api/instrument.py

```diff
@@ -17,6 +22,15 @@ def post(self):
             telescope_id, self.current_user, raise_if_none=True, mode="read"
         )
 
+        if "field_data" in data:
+            field_data = data.pop("field_data")
+            field_of_view_shape = data.pop("field_of_view_shape")
+            field_of_view_size = data.pop("field_of_view_size", None)
+
+            add_fields = True
+        else:
+            add_fields = False
```

Comment:
```suggestion
        field_data = data.pop("field_data", None)
        field_of_view_shape = data.pop("field_of_view_shape", None)
        field_of_view_size = data.pop("field_of_view_size", None)
```
paired w/ suggestion below